commands/command_checkout.go

```diff
@@ -15,6 +15,7 @@ var (
 	checkoutCmd = &cobra.Command{
 		Use:   "checkout",
 		Short: "Checks out LFS files into the working copy",
+		Long:  git_lfs_checkout_HelpText,
```

Comment:
:+1:, and along with this, I think i'd rather see `_` in the file names than `-`.